libs/components/pages/src/lib/modules/page/page-links.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { SkyLayoutHostService } from '@skyux/core';
+
+import { SkyPageLinksComponent } from './page-links.component';
+
+describe('PageLinksComponent', () => {
```

Comment:
I think we at least need a test that verifies that the component properly projects inner content via `ng-content`.

Reply:
This component is now doing even less. Do you still want an additional test?

Reply:
Yeah, I think a simple test that ensures the content appears in the component is still needed here.

Reply:
Added the test.